python/src/StudentFactory_doc.i.in

```diff
@@ -1,10 +1,9 @@
 %feature("docstring") OT::StudentFactory
 "Student factory.
 
-Available constructor:
-    StudentFactory()
-
-The parameters are estimated by a mixed method of moments and reduces MLE:
+Notes
+-----
+The parameters are estimated by a mixed method of moments and the maximum likelihood method:
```

Comment:
```suggestion
Some parameters are estimated by the method of moments and others by the maximum likelihood method:
```

Reply:
I think you forgot this suggestion. Do you agree with it?

Reply:
sorry! I had forgotten it. I modify the text.